tierkreis_visualization/frontend/src/data/api.ts

```diff
@@ -0,0 +1,51 @@
+import createFetchClient from "openapi-fetch";
+import createClient from "openapi-react-query";
+import type { paths } from "./api_stubs"; // generated by openapi-typescript
+
+const fetchClient = createFetchClient<paths>({
+  baseUrl: import.meta.env.BASE_URL,
+  headers: { Accept: "application/json" },
+});
+export const $api = createClient(fetchClient);
+
+export const fetchLogs = async (workflow_id: string) => {
+  const res = await fetchClient.GET("/api/workflows/{workflow_id}/logs", {
+    params: { path: { workflow_id } },
+    parseAs: "text",
+  });
+  return res.data ?? "No logs.";
+};
+
+export const fetchErrors = async (
+  workflow_id: string,
+  node_location_str: string
+) => {
+  const params = { path: { workflow_id, node_location_str } };
+  const res = await fetchClient.GET(
+    "/api/workflows/{workflow_id}/nodes/{node_location_str}/errors",
+    { params, parseAs: "text" }
+  );
+  return res.data ?? "No errors.";
+};
+
+export const fetchNode = async (
+  workflow_id: string,
+  node_location_str: string
+) => {
+  const params = { path: { workflow_id, node_location_str } };
+  const res = await fetchClient.GET(
+    "/api/workflows/{workflow_id}/nodes/{node_location_str}",
+    { params }
+  );
+  return res.data ?? { nodes: [], edges: [] };
+};
+
+export const listWorkflowsQuery = () => $api.useQuery("get", "/api/workflows/");
+export const logsQuery = (workflow_id: string) =>
```

Comment:
This is just convention btw but in general any function abstracting a hook also uses the `use` prefix to indicate this.
```suggestion
export const useLogsQuery = (workflow_id: string) =>
```